Turf.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "Turf"
-  s.version = "1.1.0"
+  s.version = "1.1.1"
```

Comment:
Most likely we’ll be releasing this functionality in v1.2.0, but we would take care of the version bump in a separate PR anyways. There’s likely to be a prerelease before the final v1.2.0 release.

Reply:
Reverted this